python/tvm/relay/quantize/quantize.py

```diff
@@ -195,7 +195,11 @@ def annotate_context():
     return AnnotateContext.Current
 
 
-def calibrate(graph, mod=None, ctx=None):
+def collect_stats(graph):
+    return _quantize.CollectStats(graph)
+
+
+def calibrate(graph, mod=None, ctx=None, weight_scales='power2', scales=None):
```

Comment:
Should we assume that `calibrate` should not be called by users directly and use something like `qconfig(calibrate_mode='global_scale`/`max`)?  + @tqchen